modules/images/fetchpriority/readme.txt

```diff
@@ -1,9 +1,9 @@
 === Fetchpriority ===
 
 Contributors:      wordpressdotorg
-Requires at least: 6.1
+Requires at least: 6.3
```

Comment:
Isn't this a bit pointless? Fetchpriority landed in WordPress 6.3, and this plugin does nothing when installing on 6.3. Shouldn't we just archive this plugin instead?

Reply:
Good point, I agree. Let's remove `modules/images/fetchpriority` entirely. But better to do that in a separate PR, since it's a different change. @mukeshpanchal27 Would you mind opening another one simply for removal of Fetchpriority? We don't need to ship any more updates to that plugin, and it doesn't need to be part of Performance Lab anymore.

Reply:
Left a comment on the PR, but wondering if we should reconsider removing this yet.